target/linux/apm821xx/dts/wd-mybooklive.dts

```diff
@@ -41,31 +41,33 @@
 				};
 
 				partition@1e000 {
-					compatible = "fixed-partitions";
 					reg = <0x0001e000 0x00002000>;
 					label = "u-boot-env";
+
+					compatible = "fixed-partitions";
```

Comment:
`compatible` property should be placed at the first.

https://docs.kernel.org/devicetree/bindings/dts-coding-style.html#order-of-properties-in-device-node

Reply:
OK.